libs/shared/profile-form/src/lib/shared-profile-form.tsx

```diff
@@ -122,31 +120,41 @@ export const SharedProfileForm = (props: SharedProfileFormProps) => {
             fullWidth
           />
         </Box>
-        <Box my={1}>
-          <InputLabel id="fieldOfProfessionLabel">
-            Field of Profession
-          </InputLabel>
+        <Box my={3}>
+          <TextField
+            id="description"
+            name="description"
+            multiline
+            rows={4}
+            variant="outlined"
+            label="Description"
+            inputRef={register()}
+            fullWidth
+          />
+        </Box>
+        <Box my={3}>
           <TextField
             id="fieldOfProfession"
             name="fieldOfProfession"
             type="text"
             variant="outlined"
+            label="Field of Profession"
             inputRef={register()}
             fullWidth
           />
         </Box>
-        <Box my={1}>
-          <InputLabel id="companyLabel">Company</InputLabel>
+        <Box my={3}>
           <TextField
             id="company"
             name="company"
             type="text"
             variant="outlined"
+            label="Company"
             inputRef={register()}
             fullWidth
           />
         </Box>
-        <Box my={1}>
+        <Box my={3}>
           <FormControl variant="outlined" fullWidth error={!!errors.categories}>
```

Comment:
don't use two controllers for one field FormControl is almost this same as Controller